packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -242,6 +243,7 @@ export function addRoutesToApp(app) {
   app.post('(/v[0-9]+)/indicators', createIndicators);
   app.post('(/v[0-9]+)/permissionGroups', createPermissionGroups);
   app.post('(/v[0-9]+)?/dashboardRelations', createDashboardRelations);
+  app.post('(/v[0-9]+)?/koboSync', requestManualKoBoSync);
```

Comment:
```suggestion
  app.post('(/v[0-9]+)?/syncFromService', requestManualKoBoSync);
```
And then have the query parameters take the `sync_service.code`